docs/source/en/generation_strategies.mdx

```diff
@@ -359,3 +360,26 @@ To enable assisted generation, set the `assistant_model` argument with a model.
 >>> tokenizer.batch_decode(outputs, skip_special_tokens=True)
 ['Alice and Bob are sitting in a bar. Alice is drinking a beer and Bob is drinking a']
 ```
+
+When using assisted decoding with sampling methods, the `assisted_keep_proba` argument will balance speed with
```

Comment:
I do not like this name, and I'm not completely understanding what this argument does from the doc so I can't suggest a new one😅 `assisted_threshold` maybe?